### ISSUE TITLE
Markings in feedback modal appear incorrectly scaled

### ISSUE THREAD

**Issue by lcjohnso** [zooniverse/Panoptes-Front-End#5204]
## Expected behavior

As part of the [Local Group Cluster Search](clustersearch.org) project, the Feedback Modal for drawing task should show first frame of just-completed subject, with user-submitted markers and true position of feedback trigger overlaid.

Here is a normal example:
![synclst_example](https://user-images.githubusercontent.com/2006101/51211712-fe029f80-18db-11e9-874c-35f124e1a0b4.png)

## Current behavior

Two users have report poorly scaled markers within the feedback modal when using Firefox via 
Talk ([here](https://www.zooniverse.org/projects/lcjohnso/local-group-cluster-search/talk/2154/850871) and [here](https://www.zooniverse.org/projects/lcjohnso/local-group-cluster-search/talk/2155/854939?comment=1412676))

Here's an example -- note the *huge* circle visible in lower right corner:
![badfeedbackmodal_1](https://user-images.githubusercontent.com/2006101/51211565-85034800-18db-11e9-9892-a16405009c4d.png)

## Steps to replicate

Unsure -- I have not encountered problem while using Chrome personally.  I can provide Feedback workflow and subjects for testing.

One of the users reported using Firefox 64.0 on a linux (Ubuntu) machine.

## Additional information

Seems there could be an issue with the assumed image scale within the feedback modal?

**Comment by eatyourgreens** [comment]
@lcjohnso do you have the classification that generated the feedback in the second screenshot? I'm assuming the blue circle is the marker that was placed by the volunteer? From the screenshot, it looks like the radius that was recorded in the classification was too large.

**Comment by lcjohnso** [comment]
Yes -- here is the annotations string for the classification pictured above showing four correctly recorded markings:
`[{"task":"T0","task_label":"Identify star clusters and other objects by drawing circles around them.","value":[{"r":31.78049716414141,"x":99,"y":75.36666870117188,"tool":0,"angle":-24.145541960421653,"frame":0,"details":[],"tool_label":"Star Cluster"},{"r":28.30194339616981,"x":470,"y":196.36666870117188,"tool":0,"angle":-32.005383208083494,"frame":1,"details":[],"tool_label":"Star Cluster"},{"r":30.870698080866262,"x":269,"y":88.36666870117188,"tool":0,"angle":-24.904768808095188,"frame":1,"details":[],"tool_label":"Star Cluster"},{"r":48.104053883222775,"x":297,"y":310.3666687011719,"tool":0,"angle":-20.695450734063282,"frame":1,"details":[],"tool_label":"Star Cluster"}]}]`

And here is what the modal should have (approximately) looked like:
![out-issue_fc1-b03-f15_26_color](https://user-images.githubusercontent.com/2006101/51269004-fea44000-1986-11e9-879e-448581f74af3.jpg)

It seems the problem lies in displaying those markings on the pop-up feedback modal.  But again, only on Firefox, and we've only heard from two volunteers (so could be a subset of Firefox users -- the OP was on linux/ubuntu).

**Comment by eatyourgreens** [comment]
Thanks. Those circles have radii ~30px which looks reasonable to me too. If this is only affecting two volunteers, and only on ubuntu, then we'll keep an eye on it but it isn't a high priority if it isn't breaking classifications.

**Comment by lcjohnso** [comment]
Sounds good - thanks!

**Comment by mrniaboc** [comment]
This bug is also affecting one of our REINFOCE collaborators on this project - https://www.zooniverse.org/projects/mrniaboc/reinforce-wp5-prototype/classify.
